src/app/static/src/app/actions.ts

```diff
@@ -3,14 +3,23 @@ import {RootState} from "./store";
 import {api} from "./apiService";
 import {Data, Graph} from "./generated";
 import {RootMutation} from "./mutations";
+import {DynamicFormMeta} from "@reside-ic/vue-dynamic-form";
 
 export enum RootAction {
     FetchPrevalenceGraphData = "FetchPrevalenceGraphData",
-    FetchPrevalenceGraphConfig = "FetchPrevalenceGraphConfig"
+    FetchPrevalenceGraphConfig = "FetchPrevalenceGraphConfig",
+    FetchBaselineOptions = "FetchBaselineOptions"
 }
 
 export const actions: ActionTree<RootState, RootState> = {
 
+    async [RootAction.FetchBaselineOptions](context) {
+        await api(context)
```

Comment:
Unlike the other config data, I don't think we want to freeze this response because we'll be creating clones of it which must be mutable. I think if  you copy a frozen object you get another frozen object.

Reply:
that's fine, this isn't big data anyway :+1: